.travis.yaml

```diff
@@ -0,0 +1,8 @@
+language: go
+
+script: go test -v ./...
+
+go:
+  - "1.8.x"
+  - "1.9.x"
+  - "1.10.x"
```

Comment:
You will probably also need this
https://github.com/gogo/googleapis/blob/master/install-protobuf.sh
https://github.com/gogo/googleapis/blob/master/.travis.yml#L1-L2
To also exercise the code generation again

Reply:
I will do this in a separate PR as currently the generation is tied to the third_party folder contents and I  want to be able to see that there were no generation changes.

Reply:
https://github.com/gogo/grpc-example/issues/5.